int8[] Sensordaten1
int8[] Sensordaten2
